ServerlessBenchmark/ServerlessPlatformControllers/Azure/AzureController.cs

```diff
@@ -98,21 +99,25 @@ public CloudPlatformResponse DeleteMessages(CloudPlatformRequest request)
             var response = new CloudPlatformResponse();
             var messages = request.Data[Constants.Message] as IEnumerable<string>;
             var queue = QueueClient.GetQueueReference(request.Source);
-            var tasks = new List<Task>();
-            var operationResultsByTask = new Dictionary<int, OperationContext>();
+            var operationContext = new OperationContext();
+            
+            var addMessageBlock = new ActionBlock<string>(async message =>
+            {
+                await queue.AddMessageAsync(new CloudQueueMessage(message), null, null, null, operationContext);
+            }, new ExecutionDataflowBlockOptions { SingleProducerConstrained = true, MaxDegreeOfParallelism = 32 });
```

Comment:
How does "SingleProducerConstrained" flag work with the BufferBlock? Does that mean the ActionBlock assumes that the BufferBlock will send at most 1 concurrent message?